web-app/js/portal/details/NcWmsPanel.js

```diff
@@ -324,7 +324,46 @@ Portal.details.NcWmsPanel = Ext.extend(Ext.Container, {
         var dateRangeStart = this._getDateFromPicker(this.startDateTimePicker);
         var dateRangeEnd = this._getDateFromPicker(this.endDateTimePicker);
 
-        this.dataCollection.updateNcwmsParams(dateRangeStart, dateRangeEnd, geometry);
+        this.dataCollection.setFilters(this._ncwmsParamsAsFilters(dateRangeStart, dateRangeEnd, geometry));
+    },
+
+    _ncwmsParamsAsFilters: function(dateRangeStart, dateRangeEnd, geometry) {
```

Comment:
I had considered that but as we wouldn't actually end up with a Portal.filter.Ncwms*xxx*Filter (it will use the existing Date and Geographic filters) it seemed that an anonymous object was fine in the interim.